parent/pom.xml

```diff
@@ -1084,6 +1085,89 @@
                 <artifactId>json-smart</artifactId>
                 <version>${json-smart.version}</version>
             </dependency>
+            <dependency>
+                <!-- Force fresh woodstox-core without security issue hadoop-3.3.4 -->
+                <groupId>com.fasterxml.woodstox</groupId>
+                <artifactId>woodstox-core</artifactId>
+                <version>${woodstox.version}</version>
+            </dependency>
+            <dependency>
+                <!-- Force fresh jersey-json without security issue hadoop-3.3.4 -->
+                <groupId>com.sun.jersey</groupId>
+                <artifactId>jersey-json</artifactId>
+                <version>${sun-jersey-json.version}</version>
+            </dependency>
+            <dependency>
+                <!-- Force fresh jettison without security issue hadoop-3.3.4 -->
+                <groupId>org.codehaus.jettison</groupId>
+                <artifactId>jettison</artifactId>
+                <version>${jettison.version}</version>
+            </dependency>
+            <dependency>
+                <!-- Transitive dependencies from pig-0.16 up-to-date -->
+                <groupId>tomcat</groupId>
+                <artifactId>jasper-runtime</artifactId>
+                <version>${tomcat-jasper.version}</version>
+            </dependency>
+            <dependency>
+                <!-- Transitive dependencies from pig-0.16 up-to-date -->
+                <groupId>tomcat</groupId>
+                <artifactId>jasper-compiler</artifactId>
+                <version>${tomcat-jasper.version}</version>
+            </dependency>
+            <!-- Hadoop dependencies -->
+            <dependency>
+                <groupId>org.apache.hadoop</groupId>
+                <artifactId>hadoop-common</artifactId>
+                <version>${hadoop.version}</version>
```

Comment:
@gjoranv Do we want to add exclusions to dependency management in parent pom? We'll have to reintroduce slf4j wherever we use these artifacts outside jdisc osgi context.

Reply:
No, unless there are really special circumstances. Nasty surprices will be the result.

Reply:
The special circumstance is the log4j security issue that we **never** want anywhere.